engine/swarm/services.md

```diff
@@ -80,8 +80,10 @@ flag. For more information, see
 
 When you create a service without specifying any details about the version of
 the image to use, the service uses the latest version available in Docker Hub
-or your registry. You can force the service to use a specific version of
-the image in a few different ways, depending on your desired outcome.
+or your registry, or using Notary if content trust is enabled. See the
+[note about content trust](#image_resolution_with_trust).You can force the
```

Comment:
I think the sentence about creating a service is becoming long, and mixes different concepts.

The original sentence was saying that if you omit a tag, it will be treated as `<image>:latest`. Instead of saying "the latest version", I'd change this to say "the `latest` tag", since `:latest` may not actually be the most recent version.

We also want to say that if content trust is enabled, Notary is used to authenticate the tags. This happens whether you specify a custom tag, or use the default of `latest`.

There's a missing period after the link to the note.